pkg/executor/index_merge_reader.go

```diff
@@ -395,11 +394,19 @@ func (e *IndexMergeReaderExecutor) startPartialIndexWorker(ctx context.Context,
 					SetIsStaleness(e.isStaleness).
 					SetFromSessionVars(e.Ctx().GetDistSQLCtx()).
 					SetMemTracker(e.memTracker).
-					SetPaging(e.paging).
```

Comment:
remove this useless code since `SetFromSessionVars` will set it too, and `e.paging` is set from session variable too.